examples/sglang/utils/deepseek-r1/gap_bench.sh

```diff
@@ -0,0 +1,30 @@
+MODEL=deepseek-ai/DeepSeek-R1
+OSL=5
+INPUT_FILE=<path-to-input-file>
+HEAD_PREFILL_NODE_IP=<ip>
+PORT=8000
+ARTIFACT_DIR=/benchmarks/
+
+# concurrency 1 and 2 are for warmup
+for concurrency in 1 2 8192; do
+  genai-perf profile \
+    --model ${MODEL} \
+    --tokenizer ${MODEL} \
+    --endpoint-type completions \
+    --endpoint /v1/completions \
```

Comment:
I think `v1/completions` is inferred default from `--endpoint-type completions` and can be removed